src/connect.js

```diff
@@ -19,8 +19,14 @@
  * @param {Object} baseParams
  * @return {Function}
  */
-function connect(params) {
-  // write code here
+function connect(baseParams) {
+  const baseObject = { ...baseParams };
+  return (extraParams) => {
```

Comment:
Good, but you can write simply 
```suggestion
  return (extraParams) => { ...params, ... extraParams };
```